### PR TITLE
Initial server

### PR DESCRIPTION
**Pull request by vladb55**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const app = express();
+
+app.set('views', './app/views');
+app.set('view engine', 'pug');
+app.use(express.static('public'));
+app.use(router);
+
+app.listen(3000, () => {
+    console.log('Server is started!!!');
+});
```
Comment (mvahrameev) [inline backend/app.js]: End of file

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: backend/app/controllers/index.controller.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = (request, response) => {
```
Comment (mvahrameev) [inline backend/app/controllers/index.controller.js]: Rename to `index.js`

**Comment by 102** [comment]
`backend/public/css/index.css`, `backend/public/js/index.js`, and `backend/public/images/background.png` were added, but never used. What's the reason to add them to repo? 
If you don't need these files yet, just drop them.
If you want to keep some empty folders, it's common to add empty `.gitkeep` (not sure you should do it here)

File: backend/app/views/pages/index.pug [hunk 3]
```diff
@@ -0,0 +1,8 @@
+doctype html
+html(lang="en")
+    head
+        meta(charset="utf-8")
+        meta(name="viewport", content="width=device-width, initial-scale=1")
+        title Main page
+    body
+        div Main page
```
Comment (102) [inline backend/app/views/pages/index.pug]: Still no newline

Comment (vladb55) [inline backend/app/views/pages/index.pug]: 1. I keep this folder to determine project structure

**Comment by maxmalov** [comment]
It's time to add some sense into commit messages, there is a good practice we should follow https://chris.beams.io/posts/git-commit/

Looking at this commit https://github.com/hwdtech/cms-internship/pull/9/commits/c00860fc83c200a1a575c240a7e0205c5e80d9c0 I can see that message is completely different to the actual changes

Please read the post above, review all your commit messages and fix them properly.

File: backend/app/views/pages/index.pug [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (102) [inline backend/app/views/pages/index.pug]: There's still no newline at end of file

**Pull request merged**